src/components/ProjectList/ProjectItem.js

```diff
@@ -27,6 +26,25 @@ class ProjectItem extends Component {
     this.toggleFollow = this.toggleFollow.bind(this)
   }
 
+  getTimeAgo(then) {
```

Comment:
Let's use momentjs for this, they have time zones supports as well. See `src/utils/datetime`, there is `fromNow` helper.